src/VacanciesManage/SFA.DAS.VacanciesManage.Api/Controllers/VacancyController.cs

```diff
@@ -85,6 +85,8 @@ public VacancyController (IMediator mediator, ILogger<VacancyController> logger)
                         break;
                 }
 
+                _logger.LogTrace($"Sending details to Command Handler: UKPRN:{postVacancyRequestData.User.Ukprn}");
```

Comment:
Can we remove this if possible please 😁

Reply:
obsolete code removed as not required added it for troubleshooting, but has been removed from the codebase.